src/layout/section/LeadingBrands.vue

```diff
@@ -1,15 +1,59 @@
 <template>
     <section class="max-w-5xl my-14 sm:my-28 md:my-36   mx-[auto] pt-3 px-4" >
         <h1 class="text-center font-bold mb-14 md:text-3xl text-xl">Vivamus porttitor sem et odio tempor.</h1>
-        <c-swiper>
-          <img class="bg-emerald-50" width="170" height="130">
+        <c-swiper :items="leadingBrandsData">
+          <template #default="{ items }">
+            <swiper-slide v-for="item in items" :key="item.name">
+              <img :src="item.href" :alt="item.name" class="bg-emerald-50" width="170" height="130">
+
+            </swiper-slide>
+
+          </template>
```

Comment:
`
          <template>
            <swiper-slide v-for="item in leadingBrandsData" :key="item.name">
              <img :src="item.href" :alt="item.name" class="bg-emerald-50" width="170" height="130">
            </swiper-slide>
          </template>
`
ou
            `
<swiper-slide v-for="item in leadingBrandsData" :key="item.name">
              <img :src="item.href" :alt="item.name" class="bg-emerald-50" width="170" height="130">
            </swiper-slide>
`